core/src/test/scala/org/apache/spark/sql/expression/Count0Suite.scala

```diff
@@ -21,28 +21,28 @@ import org.apache.spark.sql.BaseInitialOnceTest
 
 class Count0Suite extends BaseInitialOnceTest {
   private val countCases = Seq[String](
-    "select count(tp_int) from full_data_type_table ",
-    "select count(tp_date) from full_data_type_table ",
-    "select count(tp_tinytext) from full_data_type_table ",
-    "select count(tp_binary) from full_data_type_table ",
-    "select count(tp_decimal) from full_data_type_table ",
-    "select count(tp_timestamp) from full_data_type_table ",
-    "select count(tp_smallint) from full_data_type_table ",
-    "select count(tp_longtext) from full_data_type_table ",
-    "select count(tp_double) from full_data_type_table ",
-    "select count(tp_tinyint) from full_data_type_table ",
-    "select count(tp_mediumtext) from full_data_type_table ",
-    "select count(tp_varchar) from full_data_type_table ",
-    "select count(tp_text) from full_data_type_table ",
-    "select count(tp_float) from full_data_type_table ",
-    "select count(tp_datetime) from full_data_type_table ",
-    "select count(tp_mediumint) from full_data_type_table ",
-    "select count(tp_blob) from full_data_type_table ",
-    "select count(tp_char) from full_data_type_table ",
-    "select count(tp_nvarchar) from full_data_type_table ",
-    "select count(id_dt) from full_data_type_table ",
-    "select count(tp_real) from full_data_type_table ",
-    "select count(tp_bigint) from full_data_type_table "
+//    "select count(tp_int) from full_data_type_table ",
```

Comment:
why are these tests commented?

Reply:
It is for debugging. Will uncomment them.